apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -43,11 +66,24 @@ export function AiGeneratePage({ resLocals }: { resLocals: Record<string, any> }
                   Create question
                 </button>
               </form>
+              Or choose a question from our list of test questions:
+              <select id="user-prompt-prebuilt" onchange="setQuestionToPrebuilt()">
+                <option value=""></option>
+                ${prebuiltQuestions.map((question) => {
+                  return html`<option value="${question.question}">${question.id}</option>`;
+                })}
+              </select>
               <div id="generation-results"></div>
             </div>
           </div>
         </main>
       </body>
+      <script>
+        function setQuestionToPrebuilt() {
+          const question_set = document.getElementById('user-prompt-prebuilt').value;
+          document.getElementById('user-prompt-llm').innerHTML = question_set;
```

Comment:
Why `innerHTML` instead of `value`?

Reply:
in a `textarea`, the text displayed is its `innerHTML` for some reason. `value` doesn't have it display properly. I'm not sure why that that's the decision that the web standards committee made.

Reply:
Oh wait, no, `value` works as well; I just had a typo in naming that broke it. Changed in 4e9e66732253380bf1d73b8d0bc8872309a63541.